pkg/daemon/update.go

```diff
@@ -217,6 +220,37 @@ func addExtensionsRepo(osImageContentDir string) error {
 	return nil
 }
 
+func getOCBinaryPath() (string, error) {
+	if _, err := os.Stat(cliBinPath); err == nil {
+		return cliBinPath, nil
+	}
+
+	cliImage := "registry.svc.openshift.org/origin/4.6:cli"
+	// Pull cli image
+	args := []string{"pull", "--authfile=/var/lib/kubelet/config.json", "--quiet", cliImage}
+	if _, err := pivotutils.RunExt(numRetriesNetCommands, podmanPath, args...); err != nil {
+		return "", err
+	}
+	// Extract oc from cli Container to tmp path
+	cliTmpPath := fmt.Sprintf(cliRunBinPath)
```

Comment:
Did you mean something like:
`cliTmpPath := fmt.Sprintf("%s.tmp", cliRunBinPath)`
?